src/App.jsx

```diff
@@ -59,6 +59,30 @@ const App = () => {
                 </ProtectedRoute>
               }
             />
+            <Route
+              path="/Employees"
+              element={
+                <ProtectedRoute>
+                  <Employees />
+                </ProtectedRoute>
+              }
+            />
+            <Route
+              path="/Patients"
+              element={
+                <ProtectedRoute>
+                  <Patients />
+                </ProtectedRoute>
+              }
+            />
+            <Route
```

Comment:
I dont think there is any editing necessary in the App.jsx. It already has all setup that is needed